scripts/lp.js

```diff
@@ -0,0 +1,27 @@
+
+$("#block-yui_3_17_2_1_1530853676327_8650").appendTo(".su-form-tradie");
+$("#block-yui_3_17_2_1_1530857589866_35655").appendTo(".su-form-host");
+
+  $('.su-exit').click(function() {
```

Comment:
ideally you'd want to delegate the click handler rather than bind it directly, something like `$(document).on('click', '.su-exit', ...)`

https://learn.jquery.com/events/event-delegation/